development/compiling/compiling_for_osx.rst

```diff
@@ -26,7 +26,7 @@ Start a terminal, go to the root dir of the engine source code and type:
 
 ::
 
-    user@host:~/godot$ scons platform=osx
+    user@host:~/godot$ scons platform=osx -j$(sysctl -n hw.logicalcpu)
```

Comment:
I also wonder if we should remove the shell placeholder to make it easier to copy-paste:

```suggestion
    scons platform=osx -j$(sysctl -n hw.logicalcpu)
```

This way, the text can be clicked 3 times to select the entire line (without any extraneous text).

Reply:
Done ... I reintroduced `--jobs` as that helps for scons novices like me.

Should I squash the 3 commits?

Reply:
It's OK, I can squash them myself.